src/AccountInfoClient.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+namespace JVDS\YukiClient;
+
+use JVDS\YukiClient\Exception\Exception;
+use JVDS\YukiClient\Exception\UnexpectedTypeException;
+use JVDS\YukiClient\Soap\ClientFactory;
+use SimpleXMLElement;
+
+class AccountInfoClient extends Client
+{
+    private const WSDL = 'https://api.yukiworks.nl/ws/AccountingInfo.asmx?wsdl';
+
+    protected const SESSION_ID_ARG = 'sessionID';
+
+    public function __construct(
+        string $accessKey,
+        array $soapClientOptions = [],
+        ClientFactory $factory = null,
+        string $wsdl = self::WSDL
+    ) {
+        parent::__construct($accessKey, $wsdl, $soapClientOptions, $factory);
+    }
+
+
+    public function getTransactionDetails(
+        string $administrationID,
+        string $GLAccountCode = "",
+        string $StartDate = '2020-01-01',

```

Comment:
I use `0001-01-01` as the default value, because yuki interprets that as an optional date. Is it possible to do that here too?
See:
https://github.com/jorisvandesande/yuki-api-client/blob/master/src/ArchiveClient.php#L43

```suggestion
        string $StartDate = '0001-01-01',

```